apps/api/src/deployment/services/deployment/deployment.service.ts

```diff
@@ -193,4 +193,33 @@ export class DeploymentService {
       });
     }
   }
+
+  public async list(
+    owner: string,
+    { skip, limit }: { skip?: number; limit?: number }
+  ): Promise<{ deployments: GetDeploymentResponse["data"][]; total: number }> {
+    const deployments = await this.deploymentHttpService.loadAllDeployments(owner, "active");
+    const total = deployments.length;
+
+    const paginatedDeployments = skip !== undefined && limit !== undefined ? deployments.slice(skip, skip + limit) : deployments;
```

Comment:
I'm not sure I follow, so we load all of the deployments anyway, why paginating then? let's add params to http service?

Reply:
this also may cause event loop latency issues

Reply:
You're right, I checked the pagination proto and it support limit and offset I didn't know, I'll fix it.